charts/vaultwarden/templates/_podSpec.tpl

```diff
@@ -1,4 +1,8 @@
 {{- define "vaultwarden.podSpec" }}
+dnsConfig:
```

Comment:
Since we're modifying a default behaviour, I think it's safer if this is an optional value.